pyocd/rtos/zephyr.py

```diff
@@ -274,9 +274,12 @@ def __init__(self, target):
 
     def init(self, symbolProvider):
         # Lookup required symbols.
-        self._symbols = self._lookup_symbols(self.ZEPHYR_SYMBOLS, symbolProvider)
+        self._symbols = self._lookup_symbols(self.ZEPHYR_SYMBOLS, symbolProvider, True)
         if self._symbols is None:
             return False
+        if len(self._symbols) != len(self.ZEPHYR_SYMBOLS):
+            LOG.warning("Zephyr partially detected. Build with `CONFIG_DEBUG_THREAD_INFO=y` for full support.")
```

Comment:
Had to think a minute what it means for Zephyr to be "partially detected." Maybe more clear to say, "Zephyr kernel detected. Build your Zephyr application with `CONFIG_DEBUG_THREAD_INFO=y` to enable thread awareness."

Reply:
Thank you, I applied your suggestion :)